dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/TenantServiceImpl.java

```diff
@@ -95,6 +95,11 @@
             return result;
         }
 
+        if (!RegexUtils.isValidWindowUserName(tenantCode)) {
```

Comment:
This logic is not correct, because the username that does not satisfy Linux has been returned, and it is no longer possible to check if it satisfies the username of Windows